emojis/contributors.json

```diff
@@ -0,0 +1,21 @@
+{
+  "contributors": [
+    {"uid": "353572599957291010"},
+    {"uid": "368617749041381388"},
+    {"uid": "303732860265693185"},
+    {"uid": "406302927884386318"},
+    {"uid": "548974112131776522"},
+    {"uid": "154033306894073856"},
+    {"uid": "316765092689608706"}
```

Comment:
Comments to which they belong would be nice or using constants so you don't have to cpy them twice

Reply:
> Comments to which they belong would be nice or using constants so you don't have to cpy them twice

Initially took a name parameter with the user ID and emoji. Was asked to change this. 

I have updated to add a note to each contributor which takes the name from the emoji so that we can identify who is who.